_rules/line-height-not-important-78fd32.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 78fd32
+name: Line height in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the line height styles can be adjusted by user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][], for which the `style` attribute [declares][declared] the [line-height][] CSS property.
+
+## Expectation 1
+
+For the test target's [line-height][] property, one of the following is true:
+
+- the [declared][] value specified via the `style` attribute does not have the [important flag][]; or
+- the [computed][] value is at least `1.5` times its [computed][] [font-size][].
+
+## Assumptions
+
+There is no mechanism available on the page to adjust line height. If there is such a mechanism, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others, such as browser extensions, are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough [priority][] to override a `style` attribute with the [important flag][]. If accessibility support does not include assistive technologies that override line height through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `p` element has a `line-height` of `32px` (equals `2em` ) which is above the recommended metric, given the specified font size is `1em`.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 1em;
+		}
+	</style>
+
+	<body>
+		<p style="line-height: 2em; !important">
```

Comment:
Should this example not be 1.5em and equal to the recommended metric? Otherwise how does it differ from passed example 2 other than the unit used.